docs/Installation.md

```diff
@@ -31,6 +32,22 @@ Docker are found there.
 The rest of this document is about doing a local installation of Zonemaster-CLI,
 not relevant for running Zonemaster-CLI on Docker.
 
+## Pre-built packages
+
+Pre-built packages are available for Debian and Ubuntu (architecture amd64 only).
+
+> Note: In order to use algorithm 15 and 16 on Ubuntu 20.04 the CPAN installation
+> must be used.
+
+1) Configure Zonemaster repository
+   ```sh
+   curl -Ls https://package.zonemaster.net/setup.sh | sudo bash
```

Comment:
Maybe change `bash` to `sh`?

Reply:
> Maybe change the Prerequisites heading to Prerequisites for CPAN installation?

Sure, I can change the Prerequisites header.

> Maybe change bash to sh?

I haven't tested with sh, I suspect it will work, if it does I will update.

Reply:
I have updated with sh